homeassistant/components/plex/services.py

```diff
@@ -101,30 +104,69 @@ def get_plex_server(hass, plex_server_name=None):
     )
 
 
-def lookup_plex_media(hass, content_type, content_id):
-    """Look up Plex media for other integrations using media_player.play_media service payloads."""
-    content = json.loads(content_id)
+def process_plex_payload(
+    hass, content_type, content_id, default_plex_server=None, supports_playqueues=True
+) -> PlexMediaSearchResult:
+    """Look up Plex media using media_player.play_media service payloads."""
+    plex_server = default_plex_server
+
+    if content_id.startswith(f"{PLEX_URI_SCHEME}{{"):
```

Comment:
🙈

Reply:
Are there 2 `{` because that's how you escape it in an f-string?

Reply:
Yes.

Reply:
Perhaps this would be more readable if it _wasn't_ an f-string?

Reply:
Changed to:
```diff
-    if content_id.startswith(f"{PLEX_URI_SCHEME}{{"):
+    if content_id.startswith(PLEX_URI_SCHEME + "{"):
```